app/src/main/java/com/duckduckgo/app/di/DatabaseModule.kt

```diff
@@ -36,8 +38,9 @@ class DatabaseModule {
     @Singleton
     fun provideWebViewHttpAuthStore(
         context: Context,
+        @Named("authDbCleaner") databaseCleaner: DatabaseCleaner
```

Comment:
ditto, we can remove the named qualifier

Reply:
Done!